recipes/evalhyd-python/meta.yaml

```diff
@@ -0,0 +1,76 @@
+# name for conda package (i.e. `conda install -c conda-forge <project_name>`)
+{% set project_name = "evalhyd-python" %}
+# name for python package (i.e. `python -c "import <package_name>"`)
+{% set package_name = "evalhyd" %}
+{% set version = "0.1.0.0" %}
+{% set sha256 = "339fe203c6dea5a8d5005de3f2dbe45019d9603508c238ba01ca0d80375fe452" %}
+
+package:
+  name: {{ project_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/{{ project_name }}/-/archive/v{{ version }}/{{ project_name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script_env:
+    - CPPFLAGS=-D_LIBCPP_DISABLE_AVAILABILITY -std=c++17    # [osx]
+    - EVALHYD_PYTHON_VENDOR_XTL=FALSE
+    - EVALHYD_PYTHON_VENDOR_XTENSOR=FALSE
+    - EVALHYD_PYTHON_VENDOR_XTENSOR_PYTHON=FALSE
+    - EVALHYD_PYTHON_VENDOR_EVALHYD_CPP=FALSE
+  skip: true                                                # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.15
+    - make                                                  # [unix]
+    - ninja                                                 # [win]
+    - python
+    - pybind11
+    - setuptools
+    - pip
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - pybind11
+    - xtl >=0.7.5
+    - xtensor >=0.24.6
+    - xtensor-python >=0.26.1
+    - evalhyd-cpp ==0.1.0
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - {{ package_name }}
+  commands:
+    - python -c "import {{ package_name }}; assert {{ package_name }}.__version__ == '{{ version }}'"
+    - python -c "import {{ package_name }}; print({{ package_name }}.__doc__)"
+    - python -c "import {{ package_name }}; print({{ package_name }}.evald.__doc__)"
+    - python -c "import {{ package_name }}; print({{ package_name }}.evalp.__doc__)"
+    - python -c "import {{ package_name }}; import numpy; assert {{ package_name }}.evald(numpy.array([1, 2, 3]), numpy.array([1, 2, 3]), ['NSE'])[0][0, 0, 0] == 1"
```

Comment:
Please remove all the useless templating here and use the names that are expected (goes for everywhere else as well, but that will need to change anyway for a multi-output recipe).

A variable is useful when a value can change. The name in such a recipe cannot change, hence it just obstructs rather than helps.

Reply:
tidied these up, in the R recipe also